app/views/includes/foot.nunjucks

```diff
@@ -1,9 +1,10 @@
+{% set qs = '?v=1' %}>
```

Comment:
Rather than creating a new variable here I'd suggest re-using the existing one so as to reduce the need to update in multiple places when changes span several files. Looking at the docs for [set](https://mozilla.github.io/nunjucks/templating.html#set) this should work as long as the initial set is done at a high enough scope.